amp/repository/aim/view/viewActivityDifferences.jsp

```diff
@@ -16,6 +16,10 @@
 <link type="text/css" rel="stylesheet" href="/TEMPLATE/ampTemplate/css_2/yui_datatable.css">
 <link type="text/css" rel="stylesheet" href="/TEMPLATE/ampTemplate/css_2/desktop_yui_tabs.css">
 
+<link type="text/css" rel="stylesheet" href="/TEMPLATE/ampTemplate/css_2/activityDifference.css"> 

```

Comment:
the stylesheet is applied always, not only by clicking a print icon, so if you go to compare all now, it shows without header